agent/api/task/task_windows.go

```diff
@@ -91,6 +96,20 @@ func isBareDrive(path string) bool {
 	return false
 }
 
+func isNamedPipesPath(path string) bool {
+	matched, err := regexp.MatchString(`\\{2}\.[\\]pipe[\\].+`, path)
+
+	if err != nil {
+		return false
+	}
+
+	if matched {
```

Comment:
you can just return matched here

Reply:
Good catch on that! I've updated the code to just return matched.